content/faq/security/general.md

```diff
@@ -26,9 +26,11 @@ You can configure this through SSO using your IdP. Check with your IdP if they s
 
 ### How are sessions managed and do they expire?
 
-If configured, the IdP manages sessions.
+Docker Desktop uses tokens to manage sessions after a user signs in. Docker Desktop signs you out after 90 days, or 30 days of inactivity.
 
-Docker Desktop sessions expire after 30 days, or after 7 days of inactivity. For Docker Hub, the IdP manages sessions, if configured. If you use application-level sign-in, users are signed out due to inactivity after 14 days and must sign in again after 30 days.
+In Docker Hub, you need to re-authenticate after 24 hours. If users are authenticating using SSO, the default session timeout for the IdP is respected.
+
+Custom settings per organization for sessions aren't supported.
```

Comment:
thought: Would it make sense to put this in a callout of some kind?

Reply:
I see what you mean. We don't use callouts in the FAQ pages, but if in the future we include guidance around this in the primary manual page, could definitely see putting this in a callout!